lib/rucio/core/replica.py

```diff
@@ -1233,6 +1217,18 @@ def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns,
         yield file
         file = {}
 
+    for scope, name, bytes, md5, adler32 in _list_files_wo_replicas(files_wo_replica, session):
+        file = {
```

Comment:
not necessary to instantiate a dict here? just stream out the values

Reply:
you are right, but I did it on purpose like that. Just for readability. I don't have strong preferences, can yield directly if it is preferred.

Reply:
yeah, please, python is bad enough with memory management, no need to have extra allocations on our most often-repeated function :-)

Reply:
done